src/Components/QuestionTag.js

```diff
@@ -19,81 +41,217 @@ export default function QuestionTag() {
   };
 
   const handleAddClick = () => {
-    setInputList([...inputList, { question: "" }]);
+    setInputList([
+      ...inputList,
+      {
+        id: questionUniqueId,
+        question: "",
+        answers: [{ id: uniqueId, answer: "" }],
+      },
+    ]);
+    setUniqueId(uuidv4());
+    setQuestionUniqueId(uuidv4());
+  };
+
+  const alphabetMapping = () => {
+    let newChar = String.fromCharCode(
+      alphabet[alphabet.length - 1].charCodeAt(0) + 1
+    );
+    let newChars = alphabet.concat(newChar);
+    setAlphabet(newChars);
+  };
+  const handleAnswerAddClick = (index) => {
+    const list = [...inputList];
+    const newAnswers = [
+      ...list[index].answers,
+      { id: uniqueId, answer: "", isCorrectAnswer: false },
+    ];
+    list[index].answers = newAnswers;
+    setInputList(list);
+    setUniqueId(uuidv4());
+  };
+  const handleAnswerChange = (e, questionIndex, answerIndex) => {
+    const { name, value } = e.target;
+    const list = [...inputList];
```

Comment:
burayı da kontrol et

Reply:
kontrol ettim, üç nokta olmadan çalışmıyor çünkü oradan bir [...inputList] ile array list geliyor, newAnswers ile yeni bir answers eklendiği için yenisini ile kendisini birleştiremiyor haliyle yeni answer inputu oluşturamıyor.